src/components/text/_headers.scss

```diff
@@ -1,50 +1,39 @@
-$headerColor: $black;
-$headerColorLight: $white;
-$headerPrimaryFontFamily: $fontFamilyEmphasized;
-$headerSecondaryFontFamily: $fontFamilySecondary;
-$headerPrimaryFontSize: 64px;
-$headerPrimarySmallFontSize: 48px;
-$headerSecondaryFontSize: 16px;
-$headerSecondarySmallFontSize: 12px;
-
 $include-html: false !default;
 
 @if ($include-html) {
 
   .mint-header-primary {
     @include component;
+    @include type-variant(3, 2, 11/16, 37/16);
```

Comment:
Hm, can I fix those things in the next PR actually?
Otherwise I will have to cherry-pick a couple of commits in between branches, which is a pain in the ass TBH.